src/main/java/org/andersen/homework/util/SessionFactoryManager.java

```diff
@@ -0,0 +1,19 @@
+package org.andersen.homework.util;
+
+import lombok.Getter;
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+public class SessionFactoryManager {
+
+  @Getter
+  private static final SessionFactory sessionFactory = buildSessionFactory();
+
+  private static SessionFactory buildSessionFactory() {
+    try {
+      return new Configuration().configure("/hibernate.cfg.xml").buildSessionFactory();
```

Comment:
do we actually need hardcoded value here?

Reply:
No, redundant. Will remove it